Assets/Scripts/Interfaces/IStats.cs

```diff
@@ -0,0 +1,3 @@
+public interface IStats
+{
+}
```

Comment:
Why empty? I am guessing you are going to complete it later

Reply:
This using in the SavingSystem. For created new the stats for save.